magento/magento2
# Resolved Recently viewed products in Luma theme does not fit on scree…

### Description (*)
Using the "recently viewed products" widget on mobile will break the viewport and forcing sidescrolling.

The problem is even worse when using translated "add to cart" that make the button longer.



### Related Pull Requests

#27058 
### Fixed Issues (if relevant)

1. magento/magento2#27058 : Recently viewed products in Luma theme does not fit on screen

### Manual testing scenarios (*)

1. Add a new widget for "Recently viewed products"
2. Select Layout update "All pages" and container "Sidebar Main"
3. Visit some products in mobile view until the widget appears in mobile view

### Questions or comments


### Contribution checklist (*)
